axlearn/common/attention.py

```diff
@@ -2355,6 +2356,12 @@ class Config(BaseLayer.Config):
         # Ref: https://github.com/google/praxis/blob/main/praxis/layers/transformers.py#L1129
         # TODO (bwzhang@) Adding a unittest for the hybridnorm.
         structure: str = "prenorm"
+        # If not None, how to partition pre norm activation values.
+        prenorm_partition_spec: Optional[tuple[Optional[str]]] = None
```

Comment:
Please see my comment about `input_partition_spec` below.

Reply:
Will revise to fix this.